README.md

```diff
@@ -1 +1,171 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ToDo List] <a name="about-project"></a>
+
+**[Todo List]** Aimed at creating todo list web application. A user can apply CRUD functionality i.e, add a task, read, update and delete an already added task. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">html</a></li>
+    <li><a href="https://www.w3schools.com/css/">css</a></li>
+    <li><a href="https://www.javascripttutorial.net/">javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Home_page]**
+- **[CRUD_Functionality]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+- Install Git
+- Install NPM
+- Set A project with webpack
+- Have working browser
+- Good internet connection
+- Install a code editor like Visual Studio Code
+- Install Live Server plugging in Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Choose a directory
+- Open a Terminal
+- write: git clone https://github.com/ezraermy/ezra_portfolio/edit/portfolio_mobile_projects
+- get into to directory "portfolio_mobile_mainpage"
+- write: npm install
+
+### Install
+
+Install this project with webpack:
+- Initialize your project and install webpack
+- Add HTML to your webpack.config.js
+- Add CSS to your webpack.config.js
+- On your terminal run 'npm run dev' to make sure everything is working fine.
+
+Install this project with linters:
+
+- On your branch create .github/workflows/linters.yml
+- Do not make any changes in config files
+- Create .gitignore/ and instert node_modules/
+- On the terminal run npm install --save-dev hint@7.x
+- Copy .hintrc to the root directory of your project.
+- Run npx hint .
+- Run npx eslint .
+- Fix linter errors.
+
+### Usage
+
+To run the project, execute the following command:
+
+- Open the project by launching the Live Server plugging in visual studio code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ezra Yehuwalashet**
+
+- GitHub: [@ezraermy](https://github.com/)
+- Twitter: [@ezraermy](https://twitter.com/)
+- LinkedIn: [Ezra Yehuwalashet](https://linkedin.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Main_Page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] @ezraermy great work on your readme however the issues link is not working kindly fix it.